index.html

```diff
@@ -213,26 +225,36 @@ <h1 class="heading-about-me">About me</h1>
               <li class="flex-languages">Terminal</li>
             </ul>
           </div>
-        </div>
+        </div></div>
       </section>
       <section class="contact-us">
+        <div class="contact-us-div">
         <div class="text-contact">
           <p>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
         </div>
         <form class="input" action="https://formspree.io/f/mnqrljzr"
         method="POST">
-          <label for="fullname">
-            <input type="text" id="fullname" name="fullname" maxlength="30" placeholder="Full name" required>
+          <div class="name-container">
+          <label for="firstname">
+            <input type="text" class="item1" id="firstname" name="firstname" maxlength="30" placeholder="First Name" required>
           </label>
+          <label for="lastname">
+            <input type="text" id="lastname" class="item2" name="lastname" maxlength="30" placeholder="Last Name" required>
+          </label>
+          </div>
           <label for="email">
-            <input type="email" id="email" name="email" placeholder="Email address" required>
+            <input type="email" id="email" class="item3" name="email" placeholder="Email address" required>
           </label>
           <label for="text">
-            <textarea id="text" name="textarea" maxlength="500" placeholder="Enter text here" rows="4" cols="50" required></textarea>
+            <textarea id="text" class="item4" name="textarea" maxlength="500" placeholder="Enter text here" rows="4" cols="50" required></textarea>
           </label>
+          <div class="submit-btn">
           <button type="submit" class="get-in-touch">Get in touch</button>
+          </div>  
```

Comment:
- [ ] Kindly make sure you stick to the image as much as possible. the spacing(`margin`, `padding`). You may consider readjusting the position of this button.
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200886234-3611c43f-6f23-4310-b0a7-61105a0348b2.png)

Figma design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/200886124-00afe6e0-6422-49c6-a1f5-077f18ed1d42.png)